src/ORM/Table.php

```diff
@@ -1149,7 +1149,7 @@ public function query()
     /**
      * {@inheritDoc}
      */
-    public function updateAll($fields, $conditions)
+    public function updateAll($fields, $conditions = [])
```

Comment:
Yeah deleteAll() does not require conditions, but I thought it was because it's just faster to generate a TRUNCATE sql statement.

It just seemed to me it was kind of weird not being able to easily update *all* records of a given table with updateAll :)